packages/application-components/test-utils/test-utils.js

```diff
@@ -1,13 +1,6 @@
 import React from 'react';
 import { render } from 'react-testing-library';
-import { IntlProvider, addLocaleData } from 'react-intl';
-import en from 'react-intl/locale-data/en';
```

Comment:
React-intl does not expose locale-data anymore, as it's available from the native Intl APIs.

https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#migrate-to-using-native-intl-apis

I also don't think the polyfills are necessary for us, but if needed we can add them.